src/databricks/sql/auth/thrift_http_client.py

```diff
@@ -48,13 +48,11 @@ def __init__(
             self.scheme = parsed.scheme
             assert self.scheme in ("http", "https")
             if self.scheme == "https":
-                self.certfile = cert_file
-                self.keyfile = key_file
-                self.context = (
-                    ssl.create_default_context(cafile=cafile)
-                    if (cafile and not ssl_context)
-                    else ssl_context
-                )
+                if self._ssl_options is not None:
+                    # TODO: Not sure if those options are used anywhere - need to double-check
```

Comment:
have you double checked :P?

Reply:
Not yet - decided that it's not that critical for now. But will do - I promise 😅